.github/workflows/main.yml

```
@@ -0,0 +1,82 @@
+          poetry install --with dev
+
+  lint:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run ruff check
+        run: |
+          source .venv/bin/activate
+          ruff check
+
+  format-check: 
```

yamllint

[error] trailing spaces

(trailing-spaces)